_overviews/scala3-book/taste-contextual-abstractions.md

```diff
@@ -12,19 +12,43 @@ next-page: taste-toplevel-definitions
 TODO: Now that this is a separate section, it needs a little more content.
 {% endcomment %}
 
+Under certain circumstances, you can omit some parameters of method calls because that are considered repetitive.
 
-Under certain circumstances, the Scala compiler can “write” some parts of your programs.
-For instance, consider a program that sorts a list of addresses by two criteria, city name and then street name:
+Those parameters are called _Context Parameters_ because there are inferred by the compiler from the context surrounding the method call.
```

Comment:
The previous conversion was marked as resolved but I think you forgot to apply the suggestion :)

```suggestion
Those parameters are called _Context Parameters_ because they are inferred by the compiler from the context surrounding the method call.
```